invenio_rdm_records/fixtures/tasks.py

```diff
@@ -30,3 +32,46 @@ def create_demo_record(data):
     service = current_rdm_records.records_service
     draft = service.create(data=data, identity=system_identity)
     service.publish(id_=draft.id, identity=system_identity)
+
+
+@shared_task
+def create_demo_request(community_data, drafts_data, n_drafts=1, n_requests=1):
+    """Create n_requests requests for n_drafts drafts in a new community."""
+    # Create a demo community
+    community = current_communities.service.create(
+        system_identity,
+        community_data
+    )
+    Community.index.refresh()
+    records_service = current_rdm_records.records_service
+
+    # Create drafts in the community
+    for i in range(n_drafts):
+        _create_drafts.delay(
+            community, records_service, drafts_data[i], n_requests
+        )
+
+
+@shared_task
+def _create_drafts(community, records_service, draft_data, n_request):
+    # Create a demo draft
+    draft_data['parent'] = {
+        'review': {
+            'type': 'community-submission',
+            'receiver': {'community': community.data['uuid']}
+        }
+    }
+
+    draft = records_service.create(
+        data=draft_data, identity=system_identity
+    )
+
+    # Create requests for the draft
+    for _ in range(n_request):
+        _create_requests.delay(records_service, draft)
+
+
+@shared_task
+def _create_requests(records_service, draft):
+    # Create a demo request
+    records_service.review.submit(system_identity, draft.id)
```

Comment:
I still see the system identity here? I thought it was not correct, did something change?

Reply:
I can still see here system_identity
as we discussed yesterday it would be good to have community manager, regular user to see those requests from different points of views